### ISSUE TITLE
Satellite UI: Implement API key settings endpoints

### ISSUE THREAD

**Issue by mobyvb** [storj/storj#6783]
Blocked by https://github.com/storj/storj/issues/6782

### AC:
* When user creates a new access in the UI, call the new "create API key with settings" endpoint, and pass the same info as before, plus access type(s), permissions, bucket restrictions, and expiration
* When user shares a file, bucket, or folder in the UI, use the new "create API key with settings" to fill in details like share link and other restrictions
* When user goes into file browser, the "web browser API key" is created with "create API key with settings", with relevant details populated.
* Add an "access details" item to the three-dot-menu on the access screen, which displays the settings the access was created with, if applicable. Match the styling of "bucket details"
* If the screen size is large enough, show these additional access settings in the table view

**Comment by storjrobot** [comment]
This issue has been mentioned on **Storj Community Forum (official)**. There might be relevant details there:

https://forum.storj.io/t/backend-issues-to-work-on/27737/1

**Comment by NiaStorj** [comment]
12/19/24:

Can't be worked on until the Spanner migration is completed.

**Comment by VitaliiShpital** [comment]
closing in favor of https://github.com/storj/storj/issues/7400
